#160522376  ch-mocha-test -view-all-orders

Johnpaul-Borree opened the pull request:
#### What does this PR do?
Test api endpoint  GET /api/v1/orders
#### Description of Task to be completed?
- write tests for endpoint
- Ensure all tests pass
- look up continuous integration changes with test
#### How should this be manually tested?
- run command  npm run test
#### What are the relevant pivotal tracker stories?
#160522376 
#### Screenshots (if appropriate)
![screenshot 16](https://user-images.githubusercontent.com/25976032/45575249-c1119800-b86a-11e8-8174-c991e3f70f92.png)

Johnpaul-Borree committed:
[#160495513]tests for api endpoint create orders

- Add mocha and chai framework to node modules
- write test to create order, validate order.

Johnpaul-Borree committed:
[#160495513] update Readme badges

- Add badge for travis
- Add badge for coveralls
-Add badge for code maintainability
-Add badge for codeclimate

Johnpaul-Borree committed:
[ch-#160495513] update readme badge

Johnpaul-Borree committed:
[ch-#160495513] update readme badge

Johnpaul-Borree committed:
[ft-#160520896]Create endpoint to view all orders

- Create controller action to get all orders
- create a route for this action
- ensure the returned message is a json string

Johnpaul-Borree committed:
[ch-#160522376]Test endpoint GET /ap1/v1/orders

write tests to get all orders

KaiserPhemi commented on src/controller/ordersControllers.js:
```diff
@@ -1,37 +1,40 @@
+	res.status(200).json({status: 'Success', Orders: orders});
+}
 
 export function postOrder (req, res) {
```
#### Critical
Same feedback on #11

KaiserPhemi commented on test/ordersTests.js:
```diff
@@ -0,0 +1,93 @@
+				});
+		});
+	});
+});
```
#### Nitpick
Add a new line

Johnpaul-Borree replied:
Finished implementing feedback

KaiserPhemi reviewed (changes requested):
Work on feedback

Johnpaul-Borree committed:
[#160522376] implement feedback

- modifies .eslintrc.json for airbnb
- Refactor code
- add newline
- add spaces to objects in curly braces

Johnpaul-Borree commented:
merging view all orders mocha test to develop
